examples/bert/bert_train.py

```diff
@@ -412,7 +410,7 @@ def main(_):
     num_warmup_steps = int(
         num_train_steps * TRAINING_CONFIG["warmup_percentage"]
     )
-    epochs = TRAINING_CONFIG["epochs"]
+    epochs = min(TRAINING_CONFIG["epochs"], num_train_steps)
```

Comment:
This is a bit strange - we are comparing epochs against num_train_steps, and if epochs is over num_train_steps we set each epoch with one training_step. Maybe instead we should just set `epochs = 1`?